app/helpers/timelines_helper.rb

```diff
@@ -182,4 +182,31 @@ def push_timeline_options(timeline, target = gon)
   def timeline_options
     OpenStruct.new @timeline.options
   end
+
+  def new_timeline_link(project, &block)
+    link_to({ controller: '/timelines', action: 'new', project_id: project },
+            title: l('timelines.new_timeline'),
+            class: 'button -alt-highlight', &block)
```

Comment:
One last thing: The `block` parameter should reside on its own line, because that is what all the parameters do...

Do you agree?